RELEASE_NOTES.md

```diff
@@ -12,6 +12,8 @@
 
 ### Image Definition Changes
 
+* Introduced a dedicated FIPS mode option, adding the required packages, kernel arguments, and crypto selection
```

Comment:
When #540 lands, that functionality will need to be updated with this new flag as well.

Reply:
Got it... fixed in latest push.